pkg/chartutil/create.go

```diff
@@ -110,6 +110,7 @@ const defaultValues = `# Default values for %s.
 replicaCount: 1
 
 image:
+  registry: ""
   repository: nginx
```

Comment:
Are there a lot of charts that do this? I've seen relatively few. Most just have you adjust the respository, ie:
```
repository: myproxy/nginx
```

I don't believe that this will produce a valid reference if the registry is empty:
```
$ docker pull /mariadb:10.7
invalid reference format
```

Reply:
If you want to check for explicit registry usage, you could always just check to see that the repository has at least two `/`. There is a more complex regex that I saw somewhere, too.

If you really wanted to have a separate key for the registry, you could check to see if its empty and not add the slash.

I think that's too complex for the default generated chart though. It's important to keep it simple.